packages/bruno-app/src/components/RequestPane/QueryParams/index.js

```diff
@@ -14,6 +20,7 @@ const QueryParams = ({ item, collection }) => {
   const dispatch = useDispatch();
   const { storedTheme } = useTheme();
   const params = item.draft ? get(item, 'draft.request.params') : get(item, 'request.params');
+  const paths = item.draft ? get(item, 'draft.request.paths') : get(item, 'request.paths');
```

Comment:
I am looking into this PR as I am very interested in having this feature in Bruno, sadly in no way am I capable of reviewing JS code.
However, regarding these changes, I would fear ambiguity (in future) coming from variable names.
To avoid it, you could introduce changes similar to the following idea:
- [dir] QueryParams -> RequestParams
- [var] params -> queryParams
- [var] paths -> pathParams